Kconfig

```diff
@@ -125,11 +125,24 @@ config SECURE_BOOT
 	  This uses the public custMpk.key and certificate by default. Do not
 	  use this key in production, it is for demonstration purposes only.
 
+config RPMB_SETUP
+	bool "OPTEE RPMB setup for OTP key write"
+	depends on IMAGE_BOOT_PG2 && !SECURE_BOOT
+	help
+	   Enable CFG_RPMB_WRITE_KEY=y for OPTEE which provides
+	   OTP(one-time-programmable) RPMB key write into EMMC for initial
+	   HUK/DKEK setup for board.
```

Comment:
Do we need to put a warning here?

Reply:
Maybe rather like this: "Enable one-time pairing between processor and secure storage (RPMB on eMMC). Do not sign and distribute this version. Run it only in a secure environmenton the target device, then replace it with signed production firmware and enable secure boot." Less technical as our audience may not be familiar with all the details you mentioned.